.github/workflows/reusable.rebuild.yml

```diff
@@ -54,21 +57,21 @@ on:
         required: true
         type: string
         default: 'OFF'
-      ubuntu:
-        description: Version of Ubuntu Dockerfile to use. For example '22.04'.
+      asserts:
+        description: Build LLVM with assertions.
         required: true
         type: string
-        default: '22.04'
-      vs_generator:
-        description: VS generator to use. For example 'Visual Studio 16 2019'.
+        default: 'ON'
+      arch:
+        description: Target architecture.
         required: true
         type: string
-        default: 'Visual Studio 16 2019'
-      vs_version_str:
-        description: VS version string to use in artifact naming. For example, 'vs2019'.
+        default: 'x86'
+      ubuntu:
+        description: Version of Ubuntu Dockerfile to use. For example '22.04'.
         required: true
         type: string
-        default: 'vs2019'
+        default: '22.04'
       win_sdk:
```

Comment:
I see that `vs_version_str` has been removed. It wasn't used but I don't know why. It makes sense to have VS version configurable as Win SDK version.